App/components/Button/underButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import * as Styled from './button.styled'
+import theme from '../../theme'
+import { COLOR_KEYS } from '../../theme/colors'
+
+type Props = {
+  width?: number
+  height?: number
+  borderRadius?: number
+  shadowWidth?: number
+  shadowHeight?: number
+  shadowOpacity?: number
+  shadowColor?: keyof typeof theme.colors
+  top?: number
+}
+
+const UnderButton = ({
```

Comment:
Consider thinking of other naming variants, maybe ```UnderscoreButton``` or others you'll come up with would be more clear